src/canopy/llm/openai.py

```diff
@@ -117,10 +119,8 @@ def chat_completion(self,
         messages = [SystemMessage(content=system_message).dict()
                     ] + [m.dict() for m in chat_history]
         try:
-            response = self._client.chat.completions.create(model=self.model_name,
```

Comment:
@malexw I think it's better to pass the `model=` explicitly, for code readability.  
I'm doing a small change around this and pushing to your branch.